libdirections-hybrid/build.gradle

```diff
@@ -40,7 +40,8 @@ dependencies {
     implementation dependenciesList.kotlinStdLib
     implementation dependenciesList.coroutinesAndroid
 
-    testImplementation project(":libtesting")
+    testImplementation project(':libtesting-utils')
+    apply from: "${rootDir}/gradle/testing-dependencies.gradle"
```

Comment:
❤️

Reply:
Maybe calling it `unit-testing-dependencies`? What do you think @evabishchevich?